src/main/scala/com/cloudant/clouseau/IndexCleanupService.scala

```diff
@@ -28,6 +28,12 @@ class IndexCleanupService(ctx: ServiceContext[ConfigurationArgs]) extends Servic
       val dir = new File(rootDir, path)
       logger.info("Removing %s".format(path))
       recursivelyDelete(dir)
+    case MovePathMsg(path: String) =>
```

Comment:
I'd still prefer to see move specify a source and destination. The constructed ".recovery" path should happen in dreyfus so it can use config parameters.